samples/powerpoint/shapes/group-ungroup-shapes.yaml

```diff
@@ -0,0 +1,165 @@
+order: 6
+id: powerpoint-shapes-group-ungroup-shapes
+name: Group and ungroup shapes
+description: Shows how to create two shapes then group and ungroup them.
+author: aafvstam
+host: POWERPOINT
+api_set:
+    PowerPointApi: '1.8'
+script:
+    content: |
+        document.getElementById("group-shapes").addEventListener("click", () => tryCatch(groupShapes));
+        document.getElementById("ungroup-shapes").addEventListener("click", () => tryCatch(ungroupShapes));
+        document.getElementById("setup").addEventListener("click", () => tryCatch(setup));
+
+        async function groupShapes() {
+          await PowerPoint.run(async (context) => {
+            // Groups the geometric shapes on the current slide.
+
+            // Get the current slide.
+            context.presentation.load("slides");
+            const slide: PowerPoint.Slide = context.presentation.getSelectedSlides().getItemAt(0);
+
+            // Get the shapes.
+            slide.load("shapes");
+            await context.sync();
+
+            const shapes: PowerPoint.ShapeCollection = slide.shapes;
+            shapes.load("items/type,items/id");
+            await context.sync();
+
+            // Group the geometric shapes.
+            const shapesToGroup = shapes.items.filter((item) => item.type === PowerPoint.ShapeType.geometricShape);
+            console.log(`Number of shapes to group: ${shapesToGroup.length}`);
+            const group = shapes.addGroup(shapesToGroup);
+            group.load("id");
+            await context.sync();
+
+            console.log(`Grouped shapes. Group ID: ${group.id}`);
+          });
+        }
+
```

Comment:
Can you interact with the group programatically? If so, consider adding another method that moves the shape group with a single command.